readme.md

```diff
@@ -63,3 +63,34 @@ ecs-upload-task --file taskdefinition.yml --service hello-world-2017-05-15-10-45
 2017/11/14 18:03:48 (service hello-world-2017-05-15-10-45) has stopped 1 running tasks: (task a3e3a91b-be05-4092-bcf6-47f2075933af).
 
 ```
+
+## Creating a new multi-arch release
+
+With the uptake of M1 Macs amongst devs, we need to create builds for multiple CPU architectures to ensure compatability across all devices. This uses [`gox`](https://github.com/mitchellh/gox) to do the multi-arch builds. This can be installed by
```

Comment:
I'm wondering if this might make more sense for someone new to the project if we just explain that we're using `gox` for compilation, and call out that it also handles ARM64 builds for M1 macs. We could rename the section `Compiling binaries` or something similar. What do you think?

Reply:
Yeah thats a good point, especially because we've updated the makefile